File: python/missing.py
def missing(A):
    size = len(A)
    if size == 0:
        return 1
    total = 0
    for i in range(1, size+2):
        total +=i

    return total - sum(A)
A = [1,6,3,2,5]
print(missing(A))

File: js/missing.js
function missing(A){

    var size = A.length;
    if(size === 0)   return 1;
    var total = 0;
    var sum = 0;
    for(var j = 0; j < size; j++)
        sum += A[j];
    for(var i = 1; i <= size+1; i++)
        total += i;
    return total - sum;
}

console.log(missing([2,1]));